.github/workflows/koupleless_runtime_snapshot_2.1.x.yml

```
@@ -127,23 +127,42 @@ jobs:
             # 读取模块中的 pom.xml 来确定 JDK 版本
             jdk_version=$(grep -m 1 '<java.version>' $module/pom.xml | sed 's/<[^>]*>//g' | xargs)
             echo "${module} JDK version: ${jdk_version}"
-            # 如果是目标 jdk 版本，则执行 release 操作
 
+            # 如果是目标 jdk 版本，则记录
             if [[ "${jdk_version}" == "17" ]]; then
-              modules_in_right_jdk="${modules_in_right_jdk}${module},"
+              modules_in_right_jdk_array+=(${module})
             fi
           done
           
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)